src/components/MyKiva/MyKivaHero.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div ref="landscape" class="tw-w-full landscape tw-relative">
+		<kv-page-container class="!tw-max-w-4xl">
```

Comment:
It looks like the max-width from the design is 1072px (67rem). The design team wants to try out a new page width with this project, so let's use some custom css to apply that specific max-width. That's also going to be the max width for all the sections on the page, so it would make sense to make a reusable component to hold the page container, grid, and the custom max-width css.